src/client/components/boardDetail/index.js

```diff
@@ -40,11 +67,14 @@ class BoardDetail extends React.Component {
   generateBoardColumn() {
     let boardDisplay = [];
 
-    this.state.data.columns.forEach(function(element) {
-      boardDisplay.push(
-        <BoardColumn column={element} key={element._id} />
-      );
-    }, this);
+    if (Object.prototype.toString.call(this.state.data.columns)
```

Comment:
can it be different from an Array? if you want to check if something is an array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Reply:
good idea

Reply:
so please if `this.state.data.columns`'s type can be different from array please check it with the built in `isArray` method! :)